examples/multiple.js

```diff
@@ -1,10 +1,17 @@
-const mineflayer = require('mineflayer')
+import { createBot } from 'mineflayer'
 
-if (process.argv.length < 3 || process.argv.length > 5) {
-  console.log('Usage : node multiple.js <host> <port>')
+if (process.argv.length < 4 || process.argv.length > 6) {
+  console.log('Usage : node multiple.js <host> <port> [<name>] [<password>]')
```

Comment:
This is breaking the example... The goal of multiple.js is to show how to connect multiple bots, it makes no sense to specify only 1 name and password.